### ISSUE TITLE
Twit doesn't work in strict mode

### ISSUE THREAD

**Issue by herkyl** [ttezel/twit#202]
Running Twit in strict mode doesn't work. This mean you cannot use most of the ES6 goodies.

`iojs --use-strict path/to/index.js`

```
/node_modules/twit/lib/streaming-api-connection.js:101
        delete body
               ^^^^

SyntaxError: Delete of an unqualified identifier in strict mode.
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:413:25)
    at Object.Module._extensions..js (module.js:448:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/node_modules/twit/lib/twitter.js:9:30)
    at Module._compile (module.js:430:26)
    at Object.Module._extensions..js (module.js:448:10)
```

**Comment by herkyl** [comment]
Looking at the code I think it is pretty weird that delete is called there. It should get cleaned up by the GC anyway.

**Comment by dmeijboom** [comment]
I already opened a pull-request: https://github.com/ttezel/twit/pull/206 and I think it could be merged since it's a simple fix.

**Comment by herkyl** [comment]
Nice, you can mention "Fixes #202" in the message so this would get automatically closed

**Comment by ttezel** [comment]
Fixes #202
